src/Avalonia.Controls/SplitView.cs

```diff
@@ -0,0 +1,476 @@
+﻿using Avalonia.Controls.Primitives;
+using Avalonia.Input;
+using Avalonia.Input.Raw;
+using Avalonia.Interactivity;
+using Avalonia.Media;
+using Avalonia.Metadata;
+using Avalonia.Platform;
+using Avalonia.VisualTree;
+using System;
+using System.Reactive.Disposables;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// Defines constants for how the SplitView Pane should display
+    /// </summary>
+    public enum SplitViewDisplayMode
```

Comment:
nit: I noticed that many controls is placed in root of Avalonia.Controls folder. Should we start to move controls and related classes/enums to specific sub folders?

Reply:
I think that would be a good idea for more complex controls that are not single file... what about namespace too?

Reply:
We should keep Avalonia.Controls, because it is convenient for developers.

Reply:
I agree here, I think the controls folder could be better organized this way. Would make it easier to find things too